monai/transforms/compose.py

```diff
@@ -231,6 +231,10 @@ def randomize(self, data: Optional[Any] = None) -> None:
                     f'Transform "{tfm_name}" in Compose not randomized\n{tfm_name}.{type_error}.', RuntimeWarning
                 )
 
+    def __len__(self):
+        """Return number of transformations."""
+        return sum(len(t) if isinstance(t, Compose) else 1 for t in self.transforms)
```

Comment:
this is doing flattening and then get length, shall we have a `Compose.flatten()` and `len` just does a simple len(transform)? not sure about the specific use cases, I'm just looking at this API...

Reply:
Thanks, have added a `flatten()` method.